key-manager/src/main/scala/org/bitcoins/keymanager/WalletStorage.scala

```diff
@@ -304,6 +304,33 @@ object WalletStorage {
     }
   }
 
+  def changeAesPassword(
+      seedPath: Path,
+      oldPasswordOpt: Option[AesPassword],
+      newPasswordOpt: Option[AesPassword]): MnemonicState = {
+    decryptMnemonicFromDisk(seedPath, oldPasswordOpt) match {
```

Comment:
We should probably have a `log.info()` indicating we are attempting to change the password of the seed.